src/ProjectTemplates/Web.ProjectTemplates/content/StarterWeb-FSharp/.template.config/template.json

```diff
@@ -39,11 +39,13 @@
       "type": "parameter",
       "datatype": "bool",
       "defaultValue": "false",
+      "displayName": "Exclude Launch Settings",
       "description": "Whether to exclude launchSettings.json from the generated template."
```

Comment:
Surprisingly none of the these are needed for F-Sharp, since none of these get displayed in VIsual Studio. Framework and NoHttps options are handled specially, and all the ports properties are not shown,